src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -1045,6 +1049,20 @@ private Object getObject(Preferences prefs, String key) {
         }
     }
 
+    /**
+     * Returns a list of Strings stored by key+N with N being an incrementing number
+     */
+    private List<String> getSeries(String key) {
+        int i = 0;
+        List<String> series = new ArrayList<>();
+        String item;
+        while (!StringUtil.isBlank(item = get(key + i))) {
```

Comment:
`StringUtil#isNotBlank`?

Reply:
![grafik](https://user-images.githubusercontent.com/50491877/184963255-b976057a-773b-48f6-b6fb-937f6e4c6c0a.png)

Reply:
isNotBlank is basically a Method to be used as a MethodReferences in `.map(StringUtil::isNotBlank)` like Objects::isNull

Reply:
But still adding a literal 'not' reads better than '!'